fga-eps-mds/2020.1-stay-safe-front-end
# Associate occurrence with a location

## Issue 
Resolves #21 

## Description
Create the correct flow of occurrence register according to the prototype and with the location that user click on map.

## Checklist:



## Known problems:
The occurrences of SP and DF will be show on another map. The occurrences pinned on map are of users.

## How to validate:
Click on Report button, register an occurrence and check the occurrence pin on Home screen.
